### ISSUE TITLE
BlenderBIM - conventional way to show imperial dimensions

### ISSUE THREAD

**Issue by theoryshaw** [IfcOpenShell/IfcOpenShell#1165]
currently imperial is displayed
```3' 3-3/16"```
conventionally it's
```3' - 3 3/16"```

**Comment by Moult** [comment]
@qwiglydee since you're already going to be playing with https://github.com/IfcOpenShell/IfcOpenShell/blob/v0.6.0/src/ifcblenderexport/blenderbim/bim/helper.py#L155 this might be an easy one for you to tweak too :)

**Comment by qwiglydee** [comment]
@Moult This one isn't quite clear to me.
I'm totally not familiar with imperial notation and could not google up anything useful. Basically, I can't even parse the issue.

I'll rely on built-in blender utils for decorations.
But this issue seems to be about svg drawing export.

**Comment by Moult** [comment]
@qwiglydee yep for decorators just use the built-in Blender utils. For SVG drawing export, this function is called: https://github.com/IfcOpenShell/IfcOpenShell/blob/v0.6.0/src/ifcblenderexport/blenderbim/bim/helper.py#L155

In metric, we use a single unit, with a decimal, like 1.35m This is easy and no problem

In imperial, there is a convention to break it up into feet, inches, and fractions of an inch, where the fraction's denominator is a power of 2. That's what that helper function does. So, for example one output of the function might be `3' 3-3/16"` which a human would understand as 3 feet, and 3 inches plus 3/16 inches. It's a crazy system, I know :D 

Basically instead of separating the feet and the inches with just a space, it should instead format it separated by ` - `. Also currently the function separates the inch and the fraction with a `-`, apparently that's wrong and it should be a space instead.

Hope that makes sense.

**Comment by theoryshaw** [comment]
Yes, it's a crazy system indeed.  North America apologies. :)

**Comment by Moult** [comment]
Bump? Pin @qwiglydee does this make sense? If not, I'll make the change - it's a small change.

**Comment by qwiglydee** [comment]
@Moult I just postponed the task until "using imperial units in decorations"
I hoped the built-in formatter does support that weird syntax and I can reuse some code. But it doesn't.

I'll handle the issue soon.
